src/tribler-core/tribler_core/modules/metadata_store/restapi/tests/test_metadata_endpoint.py

```diff
@@ -252,3 +253,12 @@ def setUpPreSession(self):
 
         json_response = await self.do_request(url + '&nowait=1')
         self.assertDictEqual(json_response, {u'checking': u'1'})
+
+    @timeout(default_timeout)
+    async def test_check_torrent_query(self):
+        """
+        Test the endpoint to check if correct error message is processed if wrong timeout is provided
```

Comment:
I don't get what this description means. Please, be more clear. Something like:
"Test that the endpoint responds with an error message if the timeout parameter has a wrong value."

Reply:
I think the sentence is fine and clear.

Reply:
"if correct error message is processed" is passive voice. Turning in to active voice would read like: "something processes the correct error message". This means "something" (the system?) is doing something to the already existing "correct error message".

"To process" means "to do something to some existing thing". The endpoint does not "process" any "error messages". Instead, it processes requests. Then, it possibly *responds* with error messages.
:man_shrugging:

Reply:
ok, done